src/casp/services/api/data_manager/cellarium_general.py

```diff
@@ -0,0 +1,110 @@
+import typing as t
+
+from casp.data_manager import BaseDataManager, sql
+from casp.services import settings
+from casp.services.api import schemas
+from casp.services.api.data_manager import exceptions
+from casp.services.db import models
+
+
+class CellariumGeneralDataManager(BaseDataManager):
```

Comment:
`CellariumGeneral` -> `CAS`?

Reply:
perhaps `CellariumGeneralDAO`?

Reply:
Please explain to me why `DAO` is inaccurate in our usage?

Reply:
DAO stands for Data Access Object. Regarding the terminology, it can be used as an alternative term for Data Manager in our case. Therefore, it's important to note that we should use either DAO or Data Manager, but not both together.
DAO is a programming pattern commonly used in Java development. The current Data Manager is a solution inspired by the DAO pattern, but it doesn't strictly adhere to it, primarily because I, believe, DAO was invented during a time when data and application complexity were more primitive, and people only needed operations for creating, retrieving, updating, and destroying (CRUD) models. 
In our case, the Data Manager is a more complex application layer that enables us to perform complex filtering operations and create temporary tables in a Big Data database. It is similar to DAO in a way that it segregates data operations into a distinct layer, but it also differs from classic DAO. So, to sum up, if we call it `DAO` then it might be confusing because it is not a clear usage of DAO pattern. Just to avoid this confusion, I propose using `DataManager`.